.github/pull-request-template.md

```diff
@@ -17,3 +17,8 @@ Changes proposed in this pull request:
 
 **Related issue(s)**
 <!-- If you refer to a particular issue, provide its number. For example, `Resolves #123`, `Fixes #43`, or `See also #33`. -->
+
+**Confirm that everything was done**
+- [ ] Related issues are linked
```

Comment:
```suggestion
- [ ] PR title starts with prefix
    - feat:     A new feature
    - fix:      A bug fix
    - docs:     Documentation only changes
    - refactor: A code change that neither fixes a bug nor adds a feature
    - test:     Adding missing tests
    - chore:    Maintain. Changes to the build process or auxiliary tools/libraries/documentation
- [ ] Related issues are linked
```

Reply:
<img width="353" alt="image" src="https://github.com/kyma-project/infrastructure-manager/assets/1783008/be1f8a6a-7db6-4d09-8fef-dfc4b79a3bd7">
I've tried multiple times and I can't apply this suggestion from github. I will apply it manually.